bikeshare_app/templates/checkin.html

```diff
@@ -0,0 +1,115 @@
+        </div>
+        <form class="ui large form">
+            <h2 class="ui horizontal divider header">
+                <i class="check square icon"></i>
+                Does the bike have any damages?
+            </h2>
+
+            <div>
+                <div class="inline fields">
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageYes" name="standing" tabindex="0" value="Yes" class="hidden">
+                            <label>Yes</label>
+                        </div>
+                    </div>
+                    <div class="field">
+                        <div class="ui radio checkbox">
+                            <input type="radio" id="damageNo" name="standing" tabindex="0" value="No" class="hidden">
+                            <label>No</label>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="field" name="damageTextArea">
+                    <label>Explanation:</label>
+                    <textarea></textarea>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="lock icon"></i>
+                Was the bike locked up?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>No</label>
+                    </div>
+                </div>
+            </div>
+
+            <h2 class="ui horizontal divider header">
+                <i class="privacy icon"></i>
+                Was the key returned?
+            </h2>
+
+            <div class="inline fields">
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>Yes</label>
+                    </div>
+                </div>
+                <div class="field">
+                    <div class="ui radio checkbox">
+                        <input type="radio" name="location" tabindex="0" class="hidden">
+                        <label>No</label>
+                    </div>
+                </div>
+            </div>
+            <div class="ui fluid large bingColor submit button">Submit</div>
+
+            <div class="ui error message"></div>
+        </form>
+
+    </div>
+</div>
+{% endblock %}
+
+{% block additionalScripts %}
+<script>
+    $('div[name="damageTextArea"]').hide();
```

Comment:
Would it be better to start off as `display: none;` and then show it upon clicking the right button? Otherwise, I may, for a split second, see the textbox disappear as opposed to coming to the page never seeing the textbox.